src/components/breadcrumbs/index.md.njk

```diff
@@ -30,3 +30,11 @@ The breadcrumbs component should include the user’s current page, which should
 There are 2 ways to use the breadcrumbs component. You can use HTML or, if you are using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com), you can use the Nunjucks macro.
 
 {{ example({group: "components", item: "breadcrumbs", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
+
+### Collapsing breadcrumbs on tablet and mobile
```

Comment:
Looking at the code and [the example in Frontend](http://govuk-frontend-review.herokuapp.com/components/breadcrumbs/with-collapse-on-mobile/preview), the breadcrumbs currently only collapse on mobile – on tablet the modifier doesn't do anything.

Unsure whether it's the guidance or the implementation that's wrong.